scripts/clearwater-aio-install.sh

```diff
@@ -74,7 +74,7 @@ sed -e 's/#HEAP_NEWSIZE=".*"/HEAP_NEWSIZE="128M"/g' -i /etc/cassandra/cassandra-
 service cassandra stop
 
 # Install the remaining clearwater packages
-apt-get install -y --force-yes ellis bono restund sprout homer homestead homestead-prov
+apt-get install -y --force-yes ellis bono restund sprout homer homestead homestead-prov clearwater-prov-tools
```

Comment:
I think there's a manual command to install an AIO node in the docs somewhere - can you update this too?